tests/integration_tests/pandas_postprocessing_tests.py

```diff
@@ -256,6 +257,29 @@ def test_pivot_exceptions(self):
             aggregates={"idx_nulls": {}},
         )
 
+    def test_pivot_eliminate_cartesian_product_columns(self):
+        mock_df = DataFrame(
+            {
+                "dttm": to_datetime(["2019-01-01", "2019-01-01"]),
+                "a": [0, 1],
+                "b": [0, 1],
+                "metric": [9, np.NAN],
+            }
+        )
+
+        df = proc.pivot(
+            df=mock_df,
+            index=["dttm"],
+            columns=["a", "b"],
+            aggregates={"metric": {"operator": "mean"}},
+            drop_missing_columns=False,
+        )
+        print(df)
+        self.assertEqual(df.columns[1], "0, 0")
+        self.assertEqual(df.columns[2], "1, 1")
```

Comment:
Could we assert the full `columns` property here? Something like
```python
self.assertListEqual(df.columns.tolist(), ["__timestamp", "0, 0", "1, 1"])
```

Reply:
nice!